hanadb_exporter/exporters/prometheus_exporter.py

```diff
@@ -36,43 +37,48 @@ def __init__(self, conector):
         self._logger = logging.getLogger(__name__)
         self._hdb_connector = conector
 
-    def _execute(self, metric):
+    def _execute(self, metric_name, metric):
         """
         Create metric object
 
         Args:
             metric (dict): query, info, type structure dictionary
         """
         try:
-            value = self._hdb_connector.query(metric['query'])
-
-            if metric['type'] == core.GaugeMetricFamily:
-                metric_obj = self._manage_gauge(metric, value)
+            query_result = self._hdb_connector.query(metric['query'])
+            if metric['type'] == "gauge":
+                metric_obj = self._manage_gauge(metric_name, metric, query_result.records)
+                query_columns = []
```

Comment:
I don't understand what is this code for. `query_columns` and `metric_dict` are not used anywhere else

Reply:
This code is used later on since the pull request is not finished yet.